src/frontend/metrics/AppDiagnosticMetrics.ts

```diff
@@ -202,3 +210,27 @@ export class AppDiagnosticMetrics {
     }
   }
 }
+
+function getSelectedLanguageTag() {
+  const stored = storage.getString(LOCALE_STORAGE_KEY);
+
+  if (!stored) {
+    // TODO: Log to Sentry?
+    return null;
+  }
```

Comment:
wondering if it'd be worth logging to sentry here because it means that the storage entry doesn't exist at all, which is unexpected